_inc/client/components/settings-card/index.jsx

```diff
@@ -165,7 +167,9 @@ export const SettingsCard = props => {
 				return (
 					<JetpackBanner
 						callToAction={ upgradeLabel }
-						title={ __( 'Help your content get found and shared with SEO tools.' ) }
+						title={ __(
+							'Boost your search engine ranking with the powerful SEO tools in Jetpack Premium.'
```

Comment:
Should it also say "or Professional"?

![image](https://user-images.githubusercontent.com/426388/60743735-2312c980-9f73-11e9-80a4-85c4abdce90a.png)

Reply:
Sound sensible, I will get this updated.

```suggestion
							'Boost your search engine ranking with the powerful SEO tools in Jetpack Premium or Professional.'
```